nginx.tmpl

```diff
@@ -69,6 +70,8 @@ location {{ .Path }} {
 
 	{{ if (exists (printf "/etc/nginx/vhost.d/%s_location" .Host)) }}
 		include {{ printf "/etc/nginx/vhost.d/%s_location" .Host}};
+	{{ else if (exists (printf "/etc/nginx/vhost.d/%s_%s_location" .Host (sha1 .Path) )) }}
+		include {{ printf "/etc/nginx/vhost.d/%s_%s_location" .Host (sha1 .Path) }};
```

Comment:
This should be in order of most specific.

If you have a host-wide location, and a single path_location for one path, then that path should prioritize the path location imo

```suggestion
	{{ if (exists (printf "/etc/nginx/vhost.d/%s_%s_location" .Host (sha1 .Path) )) }}
		include {{ printf "/etc/nginx/vhost.d/%s_%s_location" .Host (sha1 .Path) }};
	{{ else if (exists (printf "/etc/nginx/vhost.d/%s_location" .Host)) }}
		include {{ printf "/etc/nginx/vhost.d/%s_location" .Host}};
```

Reply:
Current behavior means if you have a host_location, all host_path_locations are ignored.

Reply:
That makes more sense.